tests/batch/test_get_list_batches.py

```diff
@@ -0,0 +1,485 @@
+        'batch',
+        'get-list',
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+        '--reverse',
+    ])
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+
+
+def test_get_list_batches_by_head():
+    """
+    Case: get a list of batches by block identifier.
+    Expect: batches from specified block are returned.
+    """
+    head = '56100bf24eed12d2f72fe3c3ccf75fe2f53d87c224d9dda6fb98a1411070b06a' \
+           '40fcf97fccc61cb9c88442953af6ae50344ad7773f1becc6bae108443c18c551'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'batch',
+        'get-list',
+        '--head',
+        head,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+
+
+def test_get_list_batches_by_non_existing_head():
+    """
+    Case: get a list of batches by non-existing block identifier.
+    Expect: list of batches not found error message.
+    """
+    head = '56100bf24eed12d2f72fe3c3ccf75fe2f53d87c224d9dda6fb98a1411070b06a' \
+           '40fcf97fccc61cb9c88442953af6ae50344ad7773f1becc6bae108443c18c552'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'batch',
+        'get-list',
+        '--head',
+        head,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    expected_error_message = {
+        'errors': 'List of batches not found.',
+    }
+
+    assert FAILED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert dict_to_pretty_json(expected_error_message) in result.output
+
+
+@pytest.mark.parametrize('command_flag', ('--start', '--head'))
+def test_get_list_batches_with_invalid_start_head(command_flag):
+    """
+    Case: get a list of batches by invalid block identifier and batch identifier starting from.
+    Expect: the following identifier is invalid error message.
+    """
+    invalid_id = '044c7db163cf21ab9eafc9b267693e2d732411056c7530e54282946ec47cc180'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'batch',
+        'get-list',
+        command_flag,
+        invalid_id,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    expected_error_message = {
+        'errors': {
+            f'{command_flag[2:]}': [
+                f'The following identifier `{invalid_id}` is invalid.',
+            ],
+        },
+    }
+
+    assert FAILED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert dict_to_pretty_json(expected_error_message) in result.output
+
+
+def test_get_list_batches_with_limit():
+    """
+    Case: get a list of batches by limit.
+    Expect: batch is returned.
+    """
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'batch',
+        'get-list',
+        '--limit',
+        1,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+
+
+def test_get_list_batches_with_invalid_limit():
```

Comment:
You must create a test for negative limit ([example](https://github.com/Remmeauth/remme-core-cli/blob/fdedd6f35712e1c190adcd3b3b770c20f541fda7/tests/block/test_get_blocks.py#L200)) and for an invalid limit count ([example](https://github.com/Remmeauth/remme-core-cli/blob/fdedd6f35712e1c190adcd3b3b770c20f541fda7/tests/block/test_get_blocks.py#L229)).

Reply:
Done.